public/index.html

```diff
@@ -5,6 +5,20 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta http-equiv="Cache-Control" content="no-store">
     <meta http-equiv="Expires" content="0">
+    <title>Bitfinex Reporting & Performance Tools</title>
+    <meta name="description" content="Manage risk, money & strategies with our state-of-the-art pro reporting tools" />
+
+    <meta property='og:url' content='https://report.bitfinex.com/' />
+    <meta property='og:site_name' content='Bitfinex Reporting & Performance Tools' />
+    <meta property='og:type' content='website' />
+    <meta property="og:title" content="Bitfinex Reporting & Performance Tools" />
+    <meta property="og:description" content="Manage risk, money & strategies with our state-of-the-art pro reporting tools" />
+    <meta property="og:image" content="https://report.bitfinex.com/report-thumbnail-1.png">
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@bitfinex" />
+    <meta name="twitter:title" content="Bitfinex Reporting & Performance Tools" />
+    <meta name="twitter:description" content="Manage risk, money & strategies with our state-of-the-art pro reporting tools" />
+    <meta name="twitter:image" content="https://report.bitfinex.com/report-thumbnail-2.png">
```

Comment:
we can use `PUBLIC_URL` in order to support environments if it's has staging for example but I believe it's just for seo so ok